.github/workflows/gradle.yml

```
@@ -49,3 +47,18 @@ jobs:
     - name: Do one Coveralls test report
       if: matrix.os == 'ubuntu-latest' && matrix.jdk == 21
       run: ./gradlew -Dprofile=release jacocoTestReport coveralls
+  javadoc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)